### PR TITLE
Preparing release 4.5.0

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -37,18 +38,20 @@ kinto-ldap==0.3.1
 kinto-signer==2.1.1
 ldappool==2.1.0
 logging-color-formatter==1.0.2
-lxml==4.1.0
+lxml==4.1.1
 MarkupSafe==1.0
 mohawk==0.3.4
 mozilla-cloud-services-logger==1.0.1
```
Comment (leplatrem) [inline requirements.txt]: This should not be necessary, but let's leave it until the log configs are changed

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,10 +5,51 @@ This document describes changes between each past release as well as
...
+**New features**
+
+- A new custom logging formatter is available in ``kinto.core``. It fixes the issues of
+  `mozilla-cloud-services-logger <https://github.com/mozilla/mozilla-cloud-services-logger>`_.
+  Consider migrating your logging settings to :
+
+::
+
+    [formatter_json]
+    class = kinto.core.JsonLogFormatter
```
Comment (leplatrem) [inline CHANGELOG.rst]: Maybe we should take advantage of this PR to update the logging config in the example.ini

Comment (Natim) [reply CHANGELOG.rst]: Can I let you update it?

File: config/example.ini [hunk 3]
```diff
@@ -250,47 +250,34 @@ plugin = python
...
-[logger_sentry]
-level = WARN
-handlers = console
-qualname = sentry.errors
-propagate = 0
 
 [handler_console]
 class = StreamHandler
 args = (sys.stdout,)
 level = NOTSET
```
Comment (leplatrem) [inline config/example.ini]: @Natim isn't it because of this?

Comment (Natim) [reply config/example.ini]: I don't see any difference even if I put it to DEBUG

**Pull request merged**